content/en/docs/tasks/access-application-cluster/service-access-application-cluster.md

```diff
@@ -101,15 +101,12 @@ provides load balancing for an application that has two running instances.
    see the node address by running `kubectl cluster-info`. If you are
    using Google Compute Engine instances, you can use the
    `gcloud compute instances list` command to see the public addresses of your
```

Comment:
This `gcloud` command is still GCE specific but it is fine the leave it here because it is just an example of how to get public addesses.